lib/templ.ml

```diff
@@ -1,4 +1,5 @@
 open Config
+open Def
```

Comment:
it would be best to not open more module

Reply:
What is the best practice : 
- open module, call function
- call Module.function
What is the cost of opening a module?

Reply:
if `Module` is not used a lot it is better to use `Module.function`.
opening to many modules can make it confusing or hard to read

if you use a lot `Module` but in only one function, you can do a local open:
`let open Module in`

for example: https://github.com/geneweb/geneweb/blob/master/lib/util.ml#L431
we don't use `Markup` in `Util` except here, so it is best to not use a global open